src/api/courses.js

```diff
@@ -4,13 +4,93 @@ const router = require('express').Router({
 
 const { check } = require('express-validator/check')
 const { matchedData } = require('express-validator/filter')
+const moment = require('moment')
 
-const { Course, Queue, User } = require('../models')
+const { Course, Queue, Question, User, Sequelize } = require('../models')
 const { requireCourse, requireUser, failIfErrors } = require('./util')
 const requireAdmin = require('../middleware/requireAdmin')
 const requireCourseStaff = require('../middleware/requireCourseStaff')
 const safeAsync = require('../middleware/safeAsync')
 
+/**
+ * Helper for getting course question data
+ * This filters all present columns in question data returned from the sequelize query to
+ * include only the columns in our list of selectedColumns.
+ * This is to account for the case where a column may be absent or missing from questions.
```

Comment:
Curious: did you see this case while testing?

Reply:
I didn't, and I guess after thinking about it it's not too important - it would just account for if we took out a column or something (and tests would catch that).  I could take it out if that would make more sense

Reply:
Yeah, my hope is that tests would account for something like that (or that we'd catch that in code review). Iterating over every key of potentially tens of thousands of questions for a response seems to be doing a lot of unnecessary work, I think we'd be fine without this!